packages/applications-service-api/api/openapi.yaml

```diff
@@ -1006,4 +1039,97 @@ components:
           example: 1
         currentPage:
           type: number
-          example: 1
\ No newline at end of file
+          example: 1
+    SubmissionCreateRequestBody:
+      properties:
+        file:
```

Comment:
Do we want an MD5/size to be sent in the request as a way of validating the file at both ends of transfer.  Perhaps we don't.